js/gated-content/src/scss/components/_top-menu.scss

```diff
@@ -0,0 +1,86 @@
+.top-menu {
+  > div {
+    display:         flex;
+    align-items:     center;
+    justify-content: flex-end;
+
+    @include median(padding, 30px 15px, 10px);
+    @include median(max-width, 1170px, 750px, initial);
+    @include median(margin-left, auto, auto, initial);
+    @include median(margin-right, auto, auto, initial);
+
+    a:first-child {
+      margin-right: auto;
+
+      @include median(font-size, 20px, 18px);
+    }
+
+    a:not(:first-child) {
+      &:not(:last-child) {
+        padding-right: 20px;
+      }
+
+      @include median(display, block, none);
+      @include median(font-size, 16px, 12px);
+
+      svg {
+        margin-right:   10px;
+        vertical-align: text-top;
+      }
+    }
+
+    button {
+      @include median(display, none, inline-block);
+      outline:          none;
+      border:           none;
+      background-color: transparent;
+      text-align:       right;
+
+      &:before {
+        display:     inline-block;
+        font-family: 'Font Awesome 5 Free';
```

Comment:
Not sure this will work in all the OpenY themes

Reply:
replaced this icon with svg